medusa/search/core.py

```diff
@@ -443,27 +440,27 @@ def search_for_needed_episodes():
                                                                            provider=cur_provider.name)
         try:
             cur_found_results = cur_provider.search_rss(episodes)
-        except AuthException as e:
-            logger.log(u"Authentication error: " + ex(e), logger.ERROR)
+        except AuthException as error:
+            log.error(u'Authentication error: {0}', ex(error))
             continue
-        except Exception as e:
-            logger.log(u"Error while searching " + cur_provider.name + u", skipping: " + ex(e), logger.ERROR)
-            logger.log(traceback.format_exc(), logger.DEBUG)
+        except Exception as error:
+            log.error(u'Error while searching {0}, skipping: {1}', cur_provider.name, ex(error))
+            log.debug(traceback.format_exc())
```

Comment:
I think the intent here was to only show a traceback for debug purposes but always show the error.